subprojects/core/src/main/java/org/gradle/testfixtures/internal/ProjectBuilderImpl.java

```diff
@@ -83,7 +84,7 @@ public Project createProject(String name, File inputProjectDir, File gradleUserH
         GradleInternal gradle = CLASS_GENERATOR.newInstance(DefaultGradle.class, null, startParameter, topLevelRegistry.get(ServiceRegistryFactory.class));
 
         DefaultProjectDescriptor projectDescriptor = new DefaultProjectDescriptor(null, name, projectDir, new DefaultProjectDescriptorRegistry(),
-                topLevelRegistry.get(FileResolver.class));
+                topLevelRegistry.get(FileResolver.class), DefaultScriptFileResolver.empty());
```

Comment:
:x: I would avoid spreading the knowledge about `DefaultScriptFileResolver` around by making the argument optional as in f8716bb3424b76409c32ab4c3283f2d459a1e0be.